CONTRIBUTING.md

```diff
@@ -337,6 +337,25 @@ Currently, [@emilypi](https://github.com/emilypi), [@fgaz](https://github.com/fg
 `haskell.org/cabal`, and [@Mikolaj](https://github.com/Mikolaj) is the point of contact for getting
 permissions.
 
+Preview Releases
+----------------
+
+We make preview releases available to facilitate testing of development builds.
+
+Artifacts can be found on the [`cabal-head` release page](https://github.com/haskell/cabal/releases/tag/cabal-head).
+Each CI pipeline generates tarballs with a `cabal` executable, and pipelines on `HEAD` also update this release.
```

Comment:
```suggestion
The Validate CI pipeline generates tarballs with a `cabal` executable. The executable gets uploaded to this release by the pipelines that run on `master`.
```

Reply:
Thanks, that's nicer!

Reply:
I've applied and squashed it into the previous commit